byway/src/main/webapp/destinations.js

```diff
@@ -22,21 +22,21 @@ const defaultCenter = Object.freeze({
 });
 
 let userlatlng = {lat:null , lng: null};
+let map;
 
 function initializeDestinationsPage(){
-    initAutocomplete(); 
-    setProgressBar(1); 
-    fetchDestinations().then(response => {
-      updateLocations(response);
-      updateStartDestination(response);
-    });
-    
+  initAutocomplete(); 
+  setProgressBar(1); 
+  fetchDestinations().then(response => {
+    updateLocations(response,map);
```

Comment:
Since the map is global, you don't need to pass it in as a param

Reply:
done